command/cli.go

```diff
@@ -90,6 +90,9 @@ func (ba *BuildArgs) AddFlagSets(flags *flag.FlagSet) {
 	flags.Var(flagOnError, "on-error", "")
 
 	flags.BoolVar(&ba.MetaArgs.WarnOnUndeclaredVar, "warn-on-undeclared-var", false, "Show warnings for variable files containing undeclared variables.")
+
+	flags.BoolVar(&ba.ReleaseOnly, "release-only", false, "force Packer to only load plugin releases")
```

Comment:
Release-only is not clear to me that Packer is being force to load final release binaries. What do you think about  something like the following? 
```suggestion
	flags.BoolVar(&ba.ReleaseOnly, "ignore-prerelease-plugins", false, "Disable the loading of prerelease plugin binaries (x.y.z-<prerelease-name>. ")
```